frontend/src/turbo/turbo-global-listeners.ts

```diff
@@ -64,4 +64,13 @@ export function addTurboGlobalListeners() {
     activateFlashNotice();
     activateFlashError();
   });
+
+  document.addEventListener('turbo:before-morph-element', (event) => {
+    const element = event.target as HTMLElement;
+
+    // In case the element is an OpenProject custom dom element, morphing is prevented.
+    if (element.tagName.startsWith('OPCE-')) {
```

Comment:
If the HTML spec ever changes, we encounter a wonky Browser, or if we ever migrate to XHTML, the tagName might not be all-uppercase, see [MDN](https://developer.mozilla.org/en-US/docs/Web/API/Element/tagName#examples). While unlikely, I suggest ruling out this hard-to-debug edge-case and safeguard this condition by always converting the tagName to uppercase, just in case.

```suggestion
    if (element.tagName.toUpperCase().startsWith('OPCE-')) {
```